cpp/src/gandiva/precompiled/arithmetic_ops_test.cc

```diff
@@ -629,6 +629,45 @@ TEST(TestArithmeticOps, TestSignIntFloatDouble) {
   EXPECT_EQ(sign_float64(-2147483647), -1.0);
 }
 
+TEST(TestArithmeticOps, TestAbsIntFloatDouble) {
+  // abs from int32
+  EXPECT_EQ(abs_int32(43), 43);
+  EXPECT_EQ(abs_int32(-54), 54);
+  EXPECT_EQ(abs_int32(63), 63);
+  EXPECT_EQ(abs_int32(INT32_MAX), INT32_MAX);
+  // Overflow
+  // EXPECT_EQ(abs_int32(INT32_MIN), -INT32_MIN);
```

Comment:
Why do we keep this as a comment?
How about removing this entirely if this isn't needed? Or can we add an expectation instead of keeping commented out code?